src/public/modules/forms/base/directives/submit-form.directive.js

```diff
@@ -90,44 +88,29 @@ function submitFormDirective(
         $('head').append(meta)
       }
 
-      scope.formLogin = function (authType, rememberMe) {
-        if (authType === 'MyInfo') {
-          return $q
-            .when(MyInfoService.createRedirectURL(scope.form._id))
```

Comment:
`MyInfoService` seems to now be unused, should delete in this PR too

Reply:
`MyInfoService` has `validateESrvcId`, which is why i left it here cos i wanted to delete the whole file at once HAHA more satisfying but i can delete `createRedirectURL` if you want!

Reply:
delete the unused code